content.js

```diff
@@ -529,53 +530,53 @@ function reskipSponsorTime(UUID) {
     }
 }
 
-//Adds a sponsorship starts button to the player controls
-function addPlayerControlsButton() {
-    if (document.getElementById("startSponsorButton") != null) {
-        //it's already added
-        return;
-    }
-
-    let startSponsorButton = document.createElement("button");
-    startSponsorButton.id = "startSponsorButton";
-    startSponsorButton.className = "ytp-button playerButton";
-    startSponsorButton.setAttribute("title", chrome.i18n.getMessage("sponsorStart"));
-    startSponsorButton.addEventListener("click", startSponsorClicked);
-
-    let startSponsorImage = document.createElement("img");
-    startSponsorImage.id = "startSponsorImage";
-    startSponsorImage.className = "playerButtonImage";
-    startSponsorImage.src = chrome.extension.getURL("icons/PlayerStartIconSponsorBlocker256px.png");
-
-    //add the image to the button
-    startSponsorButton.appendChild(startSponsorImage);
-
-    let controls = document.getElementsByClassName("ytp-right-controls");
-    let referenceNode = controls[controls.length - 1];
-
-    if (referenceNode == undefined) {
-        //page not loaded yet
-        setTimeout(addPlayerControlsButton, 100);
-        return;
-    }
-
-    referenceNode.prepend(startSponsorButton);
-}
-
 function removePlayerControlsButton() {
-    document.getElementById("startSponsorButton").style.display = "none";
+	if (!sponsorVideoID) return;
```

Comment:
Can you fix the tabbing here and add some more new lines for spacing :)

Reply:
Ok